indico/modules/events/papers/controllers/management.py

```diff
@@ -71,3 +71,12 @@ def _process_PUT(self):
     def _process_DELETE(self):
         set_reviewing_state(self.event_new, request.view_args['reviewing_type'], False)
         return jsonify_data(flash=False, html=_render_paper_dashboard(self.event_new))
+
+
+class RHPapersAssignment(RHManagePapersBase):
+    """Assign contrubutions to reviewers and judges"""
+
+    def _process(self):
+        contribs = sorted(self.event_new.contributions, key=lambda x: x.friendly_id)
```

Comment:
Also, you could do that in the Jinja template. `event.contributions|sort(attribute='friendly_id')`